program/dashboard/Dashboard_SpreadSheetFriendly.md

```diff
@@ -0,0 +1,72 @@
+# OpenHW Project Dashboard																												
+																												
+Last Updated September 30 2022																												
+																												
+## Status Key																												
+**PC = Project Concept** 																												
+&nbsp; &#8595;  																												
+**PL = Project Launch**  																												
+&nbsp; &#8595;  																												
+**PA = Plan Approved**																												
+&nbsp; &#8595;  																												
+**PF = Project Freeze**																												
+																												
+				<br /> +																								

```

Comment:
This doesn't look right.  I _think_ you are trying to add a line break here.  If that is the case it needs to start at the beginning of a line.

Reply:
@MikeOpenHWGroup thanks, fixed. Also I think @rickoco is busy so please merge this PR if you approve. (Lots more work needs done but we need to get the new structure out there. )